portfolio.css

```diff
@@ -9,18 +9,21 @@ body {
     position: relative;
     float: left;
     width: 25%;
-    height: 200px;
-    padding: 10px;
-    /*border-radius: 50%;*/
-    /*border: 1px solid #FFF;*/
+    height: 300px;
+}
+
+.gallery {
+    overflow: auto;
+    width: 1200px;
+    margin: 0 auto;
 }
 
 .portfolio-image {
-    height: 175px;
-    width: 175px;
+    height: 225px;
+    width: 225px;
     border-radius: 20%;
     margin: 0 auto;
-    background-image: url(images/mayden-logo.png);
+    /*background-image: url(images/mayden-logo.png);*/
```

Comment:
Remember to either remove this comment or fixing the code in it when you pull into master branch.

Reply:
yep